README.md

```diff
@@ -1,58 +1,71 @@
-# unified-codes
+# Universal Codes
 
-## Getting started
+Welcome! This is Universal Codes!
 
-1. Install dependencies: `npm install`.
-2. Follow application instructions below.
+This is a responsive web application over a GraphQL API.
 
-## Applications
+It is built using Typescript, with a React Frontend and an Apollo Server with a DGraph database. The frontend code is heavily 'borrowed' from the [Notify frontend](https://github.com/openmsupply/notify/tree/main/frontend)
 
-### data-loader
+The test framework is jest; functional areas are separated into packages and managed with [lerna](https://lerna.js.org/)
 
-Application for parsing product data and populating Dgraph database.
+## Usage
 
-1. Setup Dgraph (see instructions [here](unified-codes/tools/scripts/dgraph/README.md)).
-3. Populate graph schema and demo data: `npx nx serve data-loader`.
-4. Dgraph is now running, you can access at [http://localhost:9080](http://localhost:9080).
+- Install dependencies (Using node v20+ and yarn):
+  - `yarn install`
+- Setup Dgraph (see instructions [here](unified-codes/tools/scripts/dgraph/README.md))
+  - You can then use the [explorer](https://play.dgraph.io/?latest) (server URL is http://localhost:8080)
+- Load data into DGraph:
+  - `yarn load-data`
+- Upload the GraphQL schema to DGraph:
+  - `yarn update-dgraph-schema`
+- Start the data-service:
+  - `yarn start-data-service`
+  - Access v1 api at http://localhost:4000/v1/graphql
+  - Access v2 api at http://localhost:4000/v2/graphql
 
-### data-service
+### Frontend
 
-Backend GraphQL server for accessing product data.
+We are currently re-implementing the frontend. For the existing frontend:
 
-1. Run GraphQL server: `npx nx serve data-service`.
-2. Access v1 api at http://localhost:4000/v1/graphql  
-3. Access v2 api at http://localhost:4000/v2/graphql
+1. Run web server: `npx nx serve web`
+2. Browse web at http://localhost:4200/
 
-### web 
+For the new one:
 
-Frontend for browsing and searching product data.
+1. Start the frontend: `yarn start-frontend`
 
-1. Run web server: `npx nx serve web`.
-2. Browse web at http://localhost:4200/.
+## Testing
 
-## Libraries
+- Test frontend:
+  - `yarn test-frontend`
+- TODO: test backend
 
-### data
+## Other scripts
 
-Reusable types and interfaces used by both frontend and backend code.
+- Generate GraphQL Types:
+  - `yarn generate`
 
-### ui
+## Build and release
 
-Reusable frontend React components. Component grouping based on [material-ui][https://material-ui.com/] and [atomic design](https://bradfrost.com/blog/post/atomic-web-design/) principles.
+There was the option of running the data service within Docker - this hasn't been updated with the code rearrange, going to wait to hear if it is being used first! We were building a Docker image in CI, but this has been broken for a long time, so have taken this out for now.
```

Comment:
Can be removed now...

Reply:
[45f8e61](https://github.com/msupply-foundation/unified-codes/pull/430/commits/45f8e6170810fac44545c6cd00584cf9b8279e15)